pyproject.toml

```
@@ -0,0 +1,80 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
```

It seems that spack uses `hatchling` which IIRC is also the backed recommended by PyPA, so we could consider to also use it here. Since this project is very simple to build, I think it should be as easy as just changing this section and everything else should work out of the box 
 
```suggestion
requires = ["hatchling"]  # probably add some version constraints here 
build-backend = "hatchling.build"
```